packages/fxa-auth-server/config/index.js

```diff
@@ -337,6 +337,24 @@ const conf = convict({
       format: 'port',
       doc: 'Port for Redis server',
     },
+    accessTokens: {
+      host: {
+        default: '127.0.0.1',
+        env: 'ACCESS_TOKEN_REDIS_HOST',
+        format: String,
+      },
```

Comment:
these should live on their own instance in prod, but defaults to the shared one locally